src/figure.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from figure_kind import FigureKind
+from point import Point
+
+
+@dataclass
+class Figure:
+    kind: FigureKind
+    position: Point
+    points: list[Point]
```

Comment:
please extract `list[Point]` as `Points` into `point.py`

Reply:
For what? I think `list[Point]` is clear enough

Reply:
we had a lot of `list[Point]` so please change